lib/stdlib/test/gen_statem_SUITE.erl

```diff
@@ -2368,7 +2368,7 @@ init(stop) ->
 init(stop_shutdown) ->
     {stop,shutdown};
 init(sleep) ->
-    ?t:sleep(1000),
+    ct:sleep(1000),
```

Comment:
is replacement with ct and not test_server intended? functions has a different API and also calling ct:sleep will invoke test_server:adjusted_sleep providing other logic than test_server:sleep...

Reply:
Yes, I'd like to suggest this change. If my understanding it correct, this test suite needs "at least X" milliseconds, so adjusting that X should not hurt.
BTW are there official guidelines which method to use, `receive after 1000 -> ok end`, or `timer:sleep`, or `ct:sleep`, or `test_server:sleep`?

Reply:
test_server:sleep is legacy, e.i it is from the original internal tool that now only exists as an internal part of the official common_test application. So it should not be used. I would think ct:sleep is best in the cases that there is no better alternative to sleep in a test suite as it adopts the sleep period for slower test runs such as when you run cover for instance.